services/dsp-b/src/index.ts

```diff
@@ -154,7 +154,7 @@ app.get('/interest-group.json', async (req: Request, res: Response) => {
   );
 
   res.json({
-    name: advertiser,
+    name: `dsp-b-${advertiser}`,
     owner,
 
     // x-allow-fledge: true
```

Comment:
Nit: Can this be removed?

Reply:
Actually, yes. I had that for testing purposes. Thanks for catching it.